Makefile

```diff
@@ -36,4 +36,10 @@ test: build
 			-O "./target/fungible_token-0.1.3.wasm";\
 	fi
 	@echo ──────────── Run tests ────────────────────────
-	@cargo +nightly test --release
+	@cargo test --release --package staking --test panic_test --test simple_test
+	wget https://get.gear.rs/gear-nightly-linu\x-x86_64.tar.xz && \
+	tar xvf gear-nightly-linux-x86_64.tar.xz && \
+	rm gear-nightly-linux-x86_64.tar.xz
+	@./gear --dev --tmp > /dev/null 2>&1  & echo "$$!" > gear.pid
+	cat gear.pid;
+	@cargo test --package staking --test node_test -- --test-threads=1; 	kill `(cat gear.pid)`; rm gear; rm gear.pid
```

Comment:
Please synchronize Makefile & CI configs with gear-dapps/app#11, and add the `#[ignore]` attribute to gclient tests.